snail-kotlin/src/main/java/com/compass/snail/extensions/TextViewExtensions.kt

```diff
@@ -0,0 +1,22 @@
+//  Copyright © 2017 Compass. All rights reserved.
+
+package com.compass.snail.extensions
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.TextView
+import com.compass.snail.Observable
+import com.compass.snail.Replay
+
+inline val TextView.textChanged: Observable<String>
+    get() {
+        val observable = Replay<String>(1)
+        this.addTextChangedListener(object : TextWatcher {
+            override fun beforeTextChanged(charSequence: CharSequence, i: Int, i1: Int, i2: Int) {}
+            override fun onTextChanged(charSequence: CharSequence, i: Int, i1: Int, i2: Int) {
```

Comment:
Ok we may need to create this as an instrumentation test, since we need the android context https://developer.android.com/training/testing/unit-testing/instrumented-unit-tests.html